--- orig/dan.schema.postgresql	2006-02-08 12:20:13.000000000 +0100
+++ new/dan.schema.postgresql	2006-01-28 17:05:03.000000000 +0100
@@ -40,13 +40,15 @@
 	Type			Int2, 
 	GType			Char (2), 
 	FType			Int2, 
-	Omschr			Postgres_Unknown 0x0c, 
+	Omschr			Char(222),
 	Freq			Int2, 
 	Volgorde			Int8, 
 	Opm			Int8, 
 	Opm2			Int2, 
 	SortKey			Char (240), 
-	Uitspraak			Postgres_Unknown 0x0c
+	Uitspraak		Char(222),
+	language		Char(3)
 );
 -- CREATE ANY INDEXES ...
+ALTER TABLE woordenboek ADD PRIMARY KEY (language,keyno);
 
